psi-ms.obo

```diff
@@ -21985,6 +21985,53 @@ is_a: MS:1003201 ! library provenance attribute
 xref: value-type:xsd:string "The allowed value-type for this CV term."
 relationship: has_value_type xsd\:string ! The allowed value-type for this CV term
 
+[Term]
+id: MS:1003207
+name: library creation software
+def: "Library creation software" [PSI:MS]
+is_a: MS:1000531 ! software
+is_a: MS:1003171 ! spectral library attribute
+relationship: has_value_type xsd\:string ! The allowed value-type for this CV term
+
+[Term]
+id: MS:1003208
+name: reported precursor monoisotopic m/z
+def: "The measured or inferred (as reported by the mass spectrometer acquisition software or post-processing software) of the monoisotopic peak of the precursor ion based on the MSn-1 spectrum." [PSI:MS]
```

Comment:
Isn't this redundant with `selection ion m/z`?

Reply:
This is specifically about the *monoisotopic m/z*, not whatever peak the acquisition software chooses to report (e.g. the most abundant peak or the one in the center of the isolation window). I've seen  Thermo products will report different peaks on different instrument models or with different instrument method settings, or depending upon whether or not certain bugs remain unpatched.

Reply:
The most abundant peak, often in the center of the isolation window, will usually be the isolation target m/z. How/where do you see this term being used? How do you tell whether the reported precursor m/z is monoisotopic or not without doing your own processing, which makes it no longer really the reported value? Or are you proposing to report whatever the instrument says without checking it? That would seem equivalent to selected ion m/z (which, at least by convention, will be the instrument's reported "monoisotopic" m/z if available and the isolation target m/z if not).

Reply:
This term is primarily intended for use in spectral libraries where we want to express explicitly that this is what the _library creation software_ thinks is the monoisotopic peak, how ever it is convinced. If I were making a library, I'd take the monoisotopic m/z from the output of my charge state deconvolution and deisotoping algorithm as this value, maybe averaged over time from the linked MS1 evidence. I may write that value back out to mzML, though I would still use the old term there because that's what the reader is expecting.

Reply:
In that case I'd drop "reported" from the name because that strongly implies coming from the acquisition software rather than post-processing. Perhaps "experimental" would be a better modifier? That's what mzIdentML uses (as an XML attribute).

Reply:
I think @chambm has a good point here. I think we wanted a word to differentiate from theoretical. But mzIdentML uses SpectrumIdentificationItem.experimentalMassToCharge:
```
experimentalMassToCharge  xsd:double required  The mass-to-charge value measured in 
the experiment in Daltons / charge. 
```
Curiously mzIdentML doesn't try to stipulate monoisotopic, but I suppose that is implied?
I think "experimental precursor monoisotopic m/z" is an improvement.

Reply:
I can see a convoluted way to argue that anything other than arbitrary assignment of a random value is "experimental" to a point when you use post-acquisition software to adjust the precursor monoisotopic peak assignment based on any observation from that data. I can make that change.

I don't think mzIdentML really cares about monoisotopic masses outside of modifications. That's the only time the word "monoisotopic" appears in the spec.

Reply:
It seems odd to me that "monoisotopic" should be called out specifically in the context of mass modifications but not in the context of precursors. I assumed that this was just accidental rather than intentional.

Reply:
Renamed to "experimental precursor monoisotopic m/z"